shader/fine.wgsl

```diff
@@ -264,7 +251,7 @@ fn fill_path_ms(fill: CmdFill, wg_id: vec2<u32>, local_id: vec2<u32>, result: pt
             let zp = floor(a * f32(sub_ix - 1u) + b);
             if sub_ix == 0u {
                 is_delta = y0i == xy0.y && y0i != xy1.y;
-                is_bump = xy0.x == 0.0;
+                is_bump = xy0.x == 0.0 && y0i != xy0.y;
```

Comment:
This looks good but it would be very nice to capture in the write up the nuances between `is_bump` vs `is_delta` vs `is_positive_slope` and what I believe are the vertical pixel-grid alignment checks (i.e. `y0i == xy0.y` and `y0i != xy0.y`). Maybe add a TODO here so we remember to reference it when the write up is done?

Reply:
I did a bit of explanation in comments. The logic is very, very subtle, and I think needs a real writeup, but hopefully having some explanation will be helpful until then.